### ISSUE TITLE
Allow creating a copy of a course with all exercise statements

### ISSUE THREAD

**Issue by jerhard** [ls1intum/Artemis#3765]
### Is your feature request related to a problem?

Sometimes you want to transfer exercises (without the student submissions) from a previous edition of a course to a new edition. Currently, this requires that the person that perfroms this import has instructor access to the old course, and performs the import for each exercise manually. In some cases, both of these restrictions are inconvenient. In particular this is the case when the people responsible for a course change.

### Describe the solution you'd like

There should be a way to copy all exercises from one course to a new one, so that no instructor access to the old course has to be granted to the instructor of the new course in order to allow for a (manual) exercise transfer.
In other words, this feature would allow course instructors or the Artemis admins to make such a copy of course and assign possibly other owners to it.

### Describe alternatives you've considered

An alternative would be that one allows the existing "import exercise" feature to import from some selected courses where one is not an instructor, but has some special "importer" rights. Issues with that approach are that the exercise import currently only works for programming exercises, that one cannot preview the imported exercise (at least not with a UX that is intended as a preview) and that this would introduce an additional role in the rights management of Artemis.
